tests/integration/adapters/RealAdapters.test.ts

```
@@ -45,6 +48,9 @@
     } catch (error) {
       // Ignore cleanup errors
     }
+    
+    // Clean up environment variable
+    delete process.env.AGENT_COMM_DATA_DIR;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)